### DISCUSSION TITLE
Repo language stats

### DISCUSSION THREAD

**Discussion by nxtlo** [community/community#7089] [Mobile] [answered]
Not sure if this feature has been requested or being implemented but will the stats for the used languages in the repo will be featured?

**Comment by mxie** [comment, marked as answer]
Hi @nxtlo - this feature is currently not on our roadmap. We'll reconsider it if there's a higher demand!

**Reply by nxtlo** [reply]
Thanks for the reply. Looking forward to it!

**Reply by capiet** [reply]
Yes, it is quite helpful feature, because it is available at the mobile web version, but the native apps still don’t support it. I am also looking forward to it.

**Reply by McKean** [reply]
Came here for this and want to echo that this would be a nice to have.

**Reply by nxtlo** [reply]
Same :), still waiting on this feature

**Comment by 0637371677** [comment]
> Not sure if this feature has been requested or being implemented but will the stats for the used languages in the repo will be featured?

**Comment by 0637371677** [comment]
> > Not sure if this feature has been requested or being implemented but will the stats for the used languages in the repo will be featured?
> 
>

**Comment by mehulrajdev** [comment]
Welcome to the GitHub Community, @mehulrajdev, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Reply by mehulrajdev** [reply]
mark as the answer too
